nimble/host/src/ble_uuid.c

```diff
@@ -136,6 +186,104 @@ ble_uuid_to_str(const ble_uuid_t *uuid, char *dst)
     return dst;
 }
 
+int
+ble_uuid_from_str(ble_uuid_any_t *uuid, const char *str)
+{
+    uint8_t *u8p;
+    const char *str_ptr;
+    uint32_t u16 = 0;
+    uint32_t u32 = 0;
+    uint8_t tmp_rslt = 0;
+
+    int len = (int) strlen(str);
+    str_ptr = &str[len - 2];
+
+    if (len <= BLE_UUID16_STR_MAX_LEN) {
+        uuid->u.type = BLE_UUID_TYPE_16;
+    } else if (len > BLE_UUID16_STR_MAX_LEN &&
```

Comment:
this can be simply  "else if (len <= BLE_UUID32_STR_MAX_LEN)"  cause previous check already handled <= BLE_UUID16_STR_MAX_LEN

will make code a bit easier to follow

Reply:
Done